manifest.json

```diff
@@ -47,6 +47,13 @@
         "rootDirPrefix": "html5lib-python-",
         "targetDirPrefix": "html5lib-python-"
       },
+      "Pillow": {
+        "version": "5.1.0",
+        "downloadFormat": "zip",
+        "url": "https://github.com/python-pillow/Pillow/archive/5.1.0.zip",
+        "rootDirPrefix": "Pillow-",
+        "targetDirPrefix": "Pillow-"
+      },
       "requests": {
```

Comment:
Is this package the same as the one being installed by install_third_party.sh below?

Reply:
Yes! Pillow  version no 5.1.0.

Reply:
They need to be same right?
And if we are supposed to install a third library we need to have the package related to it in manifest.json ?
I am new to this, so sorry If I am wrong somewhere.
Thanks

Reply:
`install_third_party.py` installs to oppia/third_party folder
`install_third_party.sh` 's `pip install` installs to oppia_tools folder.

Reply:
I have to use Pillow in core/domain/html_cleaner.py, in which one should I keep it then?

Reply:
Were you unable to import Pillow before making changes to manifest.json?

Reply:
I have Pillow on my machine, hence I could do that locally, but when I didn't have it in manifest.json the travis gave the following import error ---
no module named PIL.

Reply:
I recommend 
1) remove the globally installed Pillow on your machine
2) remove the Pillow folder in both oppia_tools and oppia/third_party.
3) keep trying until you can produce the same error as Travis
4) apply fix.

Reply:
i was able to produce the same error as in travis. I sent the screenshot in the mail thread. PTAL

Reply:
I meant the `no module named PIL.` error. Beside, that one is a diff topic that needs to be addressed separately.